content/v1.17/guides/extensions-release-process.md

```diff
@@ -0,0 +1,352 @@
+The supplied credentials for the remote registry require read and write access
+as upload requests to the registry specify `push` authorization scope.
+{{< /hint >}}
+
+## Quickstart: Releasing a Provider to `xpkg.crossplane.io`
+
+### Prerequisites
+
+- A GitHub repository, for example created from the
+[Upjet template](https://github.com/crossplane/upjet-provider-template)
+
+### Steps
+
+1. Create a new YAML file under `.github/workflows`. By convention, name this
+file `publish-provider-package.yaml`.
+2. Copy the following workflow definition into the file, replacing
+`<REPOSITORY NAME>` with the desired name of the repository in the registry.
+
+    ```yaml
+    name: Publish Provider Package
+
+    on:
+      workflow_dispatch:
+        inputs:
+          version:
+            description: "Version string to use while publishing the package (e.g. v1.0.0-alpha.1)"
+            default: ''
+            required: false
+          go-version:
+            description: 'Go version to use if building needs to be done'
+            default: '1.23'
+            required: false
+
+    jobs:
+      publish-provider-package:
+        uses: crossplane-contrib/provider-workflows/.github/workflows/publish-provider-non-family.yml@main
+        with:
+          repository: <REPOSITORY NAME>
+          version: ${{ github.event.inputs.version }}
+          go-version: ${{ github.event.inputs.go-version }}
+          cleanup-disk: true
+        secrets:
+          GHCR_PAT: ${{ secrets.GITHUB_TOKEN }}
+    ```
+
+3. Commit the workflow file to the default branch of the GitHub repository.
+4. The workflow should now be available to trigger via the GitHub UI in the
+`Actions` tab.
+
+### Optionally mirroring to `xpkg.upbound.io`
+
+`xpkg.upbound.io` is the registry known as the [Upbound Marketplace].
+
+To optionally push (mirror) the same package to this registry, you need
+an Upbound account.
+
+1. [Log in](https://accounts.upbound.io/login) to Upbound.
+2. Create an [access token](https://accounts.upbound.io/settings/tokens).
+3. Copy the token into a GitHub Actions secret, for example
+`XPKG_UPBOUND_TOKEN`.
+4. Reference the secret created in step 3 in the `secrets` block of the
+workflow YAML file. For example:
+
+    ```yaml
+    secrets:
+      GHCR_PAT: ${{ secrets.GITHUB_TOKEN }}
+      XPKG_UPBOUND_TOKEN: ${{ secrets.XPKG_UPBOUND_TOKEN }}
+    ```
+
+{{< hint "warning" >}}
+The process for optionally pushing to the Upbound Marketplace in this quickstart
+is changing to be consistent with how other pipelines authenticate to
+the Upbound registry.
+
+See https://github.com/crossplane-contrib/provider-workflows/issues/14 for
+details.
+{{< /hint >}}
+
+## Quickstart: Releasing a Function to `xpkg.crossplane.io`
+
+The template repository for [functions] provides a functional GitHub Action
+YAML file that pushes to `xpkg.crossplane.io` without extra configuration.
+
+To optionally configure pushing to the Upbound Marketplace (`xpkg.upbound.io`):
```

Comment:
this section needs to be removed too I think - i think the quick start for releasing a function should more focus on the steps nic outlined in the issue https://github.com/crossplane/docs/issues/785, e.g.:

```
Then, to cut a new minor version release the process is pretty simple. I:

Create a new branch named (e.g.) release-0.2 using the GitHub UI.
Create a new release named (e.g.) v0.2.0 using the GitHub UI, using the above branch. This creates a new tag.
Manually trigger the CI workflow against the new tag (e.g. v0.2.0) and tell it to tag the package with the same tag.
```